xml/operations-maintenance-database_maintenance.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="database-maintenance">
+ <title>Database Maintenance</title>
+  <para>
+  As part of Database maintenance for your cloud. See sections below for cleanup commands:
+ </para>
+<screen>
+&prompt.ardana;cat /etc/cron.daily/local-cleanup-heat
+#!/bin/bash
+su heat -s /bin/bash -c "/usr/bin/heat-manage purge_deleted -g days 14" || :
+
```

Comment:
its not clear what these commands do... or what the user would want to run them... this applies to all 3 commands here. We may want to spell out what the impact of running these commands is, and why someone would want to run them.

Reply:
I will work on this to add details to each comamnds